azure-pipelines.yml

```diff
@@ -87,47 +87,32 @@ stages:
         sudo service redis-server start
 
         # LIBYANG
-        sudo dpkg -i ../target/debs/buster/libyang*1.0.73*.deb
+        sudo dpkg -i ../target/debs/bullseye/libyang*1.0.73*.deb
       displayName: "Install dependency"
 
     - script: |
         # LIBSWSSCOMMON
         sudo apt-get -y purge libhiredis-dev libnl-3-dev libnl-route-3-dev
-        sudo dpkg -i ../target/debs/buster/libnl-3-200_*.deb
-        sudo dpkg -i ../target/debs/buster/libnl-genl-3-200_*.deb
-        sudo dpkg -i ../target/debs/buster/libnl-route-3-200_*.deb
-        sudo dpkg -i ../target/debs/buster/libnl-nf-3-200_*.deb
-        sudo dpkg -i ../target/debs/buster/libhiredis0.14_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libnl-3-200_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libnl-genl-3-200_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libnl-route-3-200_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libnl-nf-3-200_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libhiredis0.14_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libhiredis-dev_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libswsscommon_1.0.0_amd64.deb
+        sudo dpkg -i ../target/debs/bullseye/libswsscommon-dev_1.0.0_amd64.deb
+        sudo dpkg -i ../target/debs/bullseye/python3-swsscommon_1.0.0_amd64.deb
       displayName: "Install libswsscommon dependencies"
 
     - script: |
         set -ex
         # Install .NET CORE
         curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
-        sudo apt-add-repository https://packages.microsoft.com/debian/10/prod
+        sudo apt-add-repository https://packages.microsoft.com/debian/11/prod
         sudo apt-get update
         sudo apt-get install -y dotnet-sdk-5.0
       displayName: "Install .NET CORE"
 
-    - task: DownloadPipelineArtifact@2
```

Comment:
>DownloadPipelineArtifact [](http://example.com/codeflow?start=12&length=24)

The purpose of this task is to get latest sonic-swss-common artifact, even the submodule is not pointing to the latest master. It will detect unexpected new bug there. Could you check if you still need this section.